municipal_finance/models/updates.py

```diff
@@ -2,15 +2,17 @@
 from django.db import models
 
 from django.contrib.auth.models import User
+from django.utils.deconstruct import deconstructible
 
 
-def upload_to(path):
-    def get_file_path(instance, filename):
+@deconstructible
+class UpdateFilePath(object):
```

Comment:
why not just a plain old function?

Reply:
There was a point when migrations required a class to serialise field, but it's no longer needed. Thought I'd keep it that way for now till I know it's not needed again.